doc/Language/faq.pod6

```diff
@@ -677,7 +677,7 @@ about the object deemed unimportant to understanding the essence of the object.
 
 Or phrased differently, C<$obj.Str> gives a string representation,
 C<$obj.gist> provides a short summary of that object suitable for
-fast recognition by a human, and C<$obj.perl> gives a Perlish representation
+fast recognition by a human, and C<$obj.perl> gives a Rakuish representation
```

Comment:
Perhaps mention the `.raku` method instead of `.perl` ?

Reply:
I thought about it, but it's not documented yet. Guess mentioning can't hurt.

Reply:
Then we should think about documenting `.raku`, as 2019.11 does have it.

Reply:
@lizmat think 8b0b7f1 resolves this?